conf_files/pocs.yaml

```diff
@@ -66,4 +90,4 @@ messaging:
 #         webhook_url: [your_webhook_url]
 #         output_timestamp: False
 
-state_machine: simple_state_table
+state_machine: simple_state_table
```

Comment:
Can we keep the newlines at the ends of files?

Reply:
Sorry, I think my autopep8 is doing that. I noticed a lot of those changes. I'll try to fix.